src/lib/setup.ts

```diff
@@ -1,4 +1,4 @@
-import ddTracer from "dd-trace"
+import * as ddTracer from "dd-trace"
```

Comment:
Hmm wonder why this didn't display an error by the editor. Good catch though.

Reply:
it's because we're not using webpack. There's an option (`allowSyntheticDefaultImports`) in tsconfig.json that allows default imports in typescript but it doesn't fix the compiled JS import

https://www.typescriptlang.org/docs/handbook/compiler-options.html